Tests/RelayerTests/AuthTests/ED25519DIDKeyTests.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import XCTest
+@testable import WalletConnectRelay
+
+final class ED25519DIDKeyFactoryTests: XCTestCase {
+    let expectedDid = "did:key:z6MkodHZwneVRShtaLf8JKYkxpDGp1vGZnpGmdBpX8M2exxH"
+    var sut: ED25519DIDKeyFactoryImpl!
+
+    func test() {
+
```

Comment:
planning to implement?

Reply:
implemented but the factory itself is not implemented - scope of #288